--- a/include/drm/ttm/ttm_bo_api.h	2021-03-21 09:41:37.000000000 +0800
+++ b/include/drm/ttm/ttm_bo_api.h	2021-03-24 12:04:01.029547250 +0800
@@ -612,8 +612,11 @@
 static inline void ttm_bo_unpin(struct ttm_buffer_object *bo)
 {
 	dma_resv_assert_held(bo->base.resv);
-	WARN_ON_ONCE(!bo->pin_count);
-	--bo->pin_count;
+        WARN_ON_ONCE(!kref_read(&bo->kref));
+        if (bo->pin_count)
+		--bo->pin_count;
+        else
+        	WARN_ON_ONCE(true);
 }
 
 int ttm_mem_evict_first(struct ttm_bo_device *bdev,
