--- main.cpp	2018-07-21 15:20:53.000000000 +0800
+++ main.cpp	2018-07-21 21:40:23.430689206 +0800
@@ -481,6 +481,28 @@
 
     prepareSignalHandler();
 
+    mainWindow->setStyleSheet(QString::fromUtf8("QScrollBar:vertical {"
+    "    border: 1px solid #999999;"
+    "    background:white;"
+    "    min-width:20px;    "
+    "    margin: 0px 0px 0px 0px;"
+    "}"
+    "QScrollBar::handle:vertical {"
+    "    background: rgb(180, 180, 240);"
+    "    min-height: 50px;"
+    "}"
+    "QScrollBar::add-line:vertical {"
+    "    height: 0px;"
+    "    subcontrol-position: bottom;"
+    "    subcontrol-origin: margin;"
+    "}"
+    "QScrollBar::sub-line:vertical {"
+    "    height: 0 px;"
+    "    subcontrol-position: top;"
+    "    subcontrol-origin: margin;"
+    "}"
+    ));
+
     mainWindow->show();
     mainWindow->setAttribute(Qt::WA_DeleteOnClose);
 
