--- mainwindow.cpp	2018-07-21 15:20:53.000000000 +0800
+++ mainwindow.cpp	2018-07-22 16:21:06.430552535 +0800
@@ -2898,10 +2898,10 @@
 
         if(s_settings->mainWindow().currentPageInWindowTitle())
         {
-            currentPage = QString(" (%1 / %2)").arg(tab->currentPage()).arg(tab->numberOfPages());
+            currentPage = QString("(%1/%2) ").arg(tab->currentPage()).arg(tab->numberOfPages());
         }
 
-        tabText = m_tabWidget->currentTabText() + currentPage + QLatin1String("[*] - ");
+        tabText = currentPage + m_tabWidget->currentTabText() + QLatin1String("[*] - ");
     }
 
     const QString instanceName = qApp->objectName();
