diff --git a/src/yoga-bios-unlock.c b/src/yoga-bios-unlock.c
index fa3e5ca..b5efb53 100644
--- a/src/yoga-bios-unlock.c
+++ b/src/yoga-bios-unlock.c
@@ -22,11 +22,11 @@
 #define __BOARD_NAME "LNVNB161216"
 #define __BOARD_VENDOR "LENOVO"
 #define __BOARD_VERSION_88 "SDK0J40688 WIN  "
-#define __BOARD_VERSION_00 "SDK0J40700 WIN  "
+#define __BOARD_VERSION_00 "SDK0L77769 WIN  "
 #define __BOARD_VERSION_09 "SDK0J40709 WIN  "
 #define __BOARD_VERSION_26 "SDK0Q55726 WIN  "
 
-#define __CHASSIS_VERSION "Yoga Slim 7 14ARE05"
+#define __CHASSIS_VERSION "Yoga 14sARE 2020"
 
 typedef struct dmi_strings dmi_strings_t;
 struct dmi_strings {
