diff --git a/networkd-notify b/networkd-notify
index 4fdc71d..3943be6 100755
--- a/networkd-notify
+++ b/networkd-notify
@@ -17,8 +17,7 @@ import dbus.mainloop.glib
 
 
 # Constants
-NETWORKCTL = ['/usr/bin/networkctl', '/bin/networkctl']
-IWCONFIG = ['/usr/bin/iwconfig', '/sbin/iwconfig']
+NETWORKCTL = '/usr/bin/networkctl'
 APP_NAME  = 'networkd'
 
 STATE_IGN = {'carrier', 'degraded'}
@@ -29,8 +28,8 @@ STATE_MAP = {'off': 'offline',
 
 # Nifty globals
 IFACE_MAP = {}
-NOTIFY_MS_SHORT = 1000
-NOTIFY_MS_LONG = 3000
+NOTIFY_MS_SHORT = 5000
+NOTIFY_MS_LONG = 10000
 NOTIFY_IF = None
 
 
@@ -46,13 +45,6 @@ def notify(title, text, time=NOTIFY_MS_SHORT):
         NOTIFY_IF.Notify(APP_NAME, 0, '', title, text, '', '', time)
 
 
-def resolve_path(path_list):
-    for path in path_list:
-        if os.path.exists(path):
-            return path
-    return None
-
-
 def update_iface_map():
     out = subprocess.check_output([NETWORKCTL, 'list', '--no-pager', '--no-legend'])
     IFACE_MAP.clear()
@@ -95,10 +87,8 @@ def unquote(buf, char='\\'):
 
 
 def get_wlan_essid(iface):
-    out = subprocess.check_output([IWCONFIG, '--', iface])
-    line = out.split(b'\n')[0].decode('ascii')
-    essid = line[line.find('ESSID:')+7:-3]
-    return unquote(essid)
+    essid = subprocess.check_output(f"/usr/bin/iw {iface} info | /usr/bin/grep -Po '(?<=ssid ).*'", shell=True)
+    return unquote(essid.decode(sys.stdout.encoding).strip())
 
 
 def property_changed(typ, data, _, path):
@@ -154,14 +144,6 @@ if __name__ == '__main__':
     ap = argparse.ArgumentParser(description='networkd notification daemon')
     args = ap.parse_args()
 
-    NETWORKCTL = resolve_path(NETWORKCTL)
-    if NETWORKCTL is None:
-        sys.exit("networkctl binary not found")
-
-    IWCONFIG = resolve_path(IWCONFIG)
-    if IWCONFIG is None:
-        sys.exit("iwconfig binary not found")
-
     # listen on system-wide bus for networkd events
     dbus.mainloop.glib.DBusGMainLoop(set_as_default=True)
     bus = dbus.SystemBus()
