diff --git a/render.c b/render.c
index ebf8bfb..e8a01c1 100644
--- a/render.c
+++ b/render.c
@@ -305,8 +305,7 @@ void render_frame(struct swaylock_surface *surface) {
 			double x, y;
 			cairo_text_extents(cairo, text, &extents);
 			cairo_font_extents(cairo, &fe);
-			x = (buffer_width / 2) -
-				(extents.width / 2 + extents.x_bearing);
+			x = (buffer_width / 2) - (extents.x_advance / 2);
 			y = (buffer_diameter / 2) +
 				(fe.height / 2 - fe.descent);
 
@@ -327,8 +326,7 @@ void render_frame(struct swaylock_surface *surface) {
 
 			cairo_text_extents(cairo, text_l1, &extents_l1);
 			cairo_font_extents(cairo, &fe_l1);
-			x_l1 = (buffer_width / 2) -
-				(extents_l1.width / 2 + extents_l1.x_bearing);
+			x_l1 = (buffer_width / 2) - (extents_l1.x_advance / 2);
 			y_l1 = (buffer_diameter / 2) +
 				(fe_l1.height / 2 - fe_l1.descent) - arc_radius / 10.0f;
 
@@ -342,8 +340,7 @@ void render_frame(struct swaylock_surface *surface) {
 			cairo_set_font_size(cairo, arc_radius / 6.0f);
 			cairo_text_extents(cairo, text_l2, &extents_l2);
 			cairo_font_extents(cairo, &fe_l2);
-			x_l2 = (buffer_width / 2) -
-				(extents_l2.width / 2 + extents_l2.x_bearing);
+			x_l2 = (buffer_width / 2) - (extents_l2.x_advance / 2);
 			y_l2 = (buffer_diameter / 2) +
 				(fe_l2.height / 2 - fe_l2.descent) + arc_radius / 3.5f;
 
