diff --git a/src/hstr_history.c b/src/hstr_history.c
index 8dcc86b..74d6702 100644
--- a/src/hstr_history.c
+++ b/src/hstr_history.c
@@ -403,6 +403,6 @@ int history_mgmt_remove_from_ranked(char *cmd, HistoryItems *history) {
 void history_mgmt_flush(void)
 {
     if(dirty && !isZshParentShell()) {
-        fill_terminal_input("history -r\n", false);
+        // fill_terminal_input("history -r\n", false);
     }
 }
