Description:Fix string format error
Author: Daniel Echeverri <epsilon@debian.org>
Forwarded: https://github.com/dvorka/hstr/issues/434
Last-Update: <2021-11-08>
--- a/src/hstr.c
+++ b/src/hstr.c
@@ -740,7 +740,7 @@
         color_attr_on(COLOR_PAIR(HSTR_COLOR_INFO));
         color_attr_on(A_BOLD);
     }
-    mvprintw(hstr->promptYNotification, 0, screenLine);
+    mvprintw(hstr->promptYNotification, 0,"%s", screenLine);
     if(hstr->theme & HSTR_THEME_COLOR) {
         color_attr_off(A_BOLD);
         color_attr_on(COLOR_PAIR(HSTR_COLOR_NORMAL));
